RealEstate/apps/core/forms.py

```diff
@@ -2,10 +2,21 @@
 from django.contrib.auth.forms import UserCreationForm
 from django.core.exceptions import ValidationError
 
-from RealEstate.apps.core.models import User
+from RealEstate.apps.core.models import User, Category
 from RealEstate.apps.pending.models import PendingHomebuyer
 
 
+class AddCategoryForm(forms.Form):
+    summary = forms.CharField(max_length=100)
+    description = forms.CharField(required=False, max_length=200)
+    
+
+class EditCategoryForm(forms.Form):
+    edit_summary = forms.CharField(label='Summary', max_length=100)
+    edit_description = forms.CharField(required=False, label='Description', max_length=200)
+    catID = forms.CharField(widget=forms.HiddenInput())
```

Comment:
Similar to a comment last week, this can be refactored in order to ensure all relevant data is defined in the model, and the forms can be derived from that.  Django uses ModelForms to construct a form based on model field definitions.  So for example, it would look at the Category model and create a required summary field with a max length of 128.

```python
class AddCategoryForm(forms.ModelForm):
    class Meta:
        model = Category
        fields = ('summary', 'description')

class EditCategoryForm(forms.ModelForm):
    catID = forms.CharField(widget=forms.HiddenInput())
    class Meta:
        model = Category
        fields = ('summary', 'description')